.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
```

Comment:
id:# is one of the supported formats of the TP/Github integration so I went with that over something like TP-#

Reply:
just spitballing:
```
description

Target Process Story: [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
```

Reply:
sure, that works too

Reply:
maybe drop the "Target Process Story:" part? seems redundant to me

Reply:
Works for me, it's similar enough to what we do in the draft repo https://github.com/NCARB/Draft/wiki/Pull-Request-Review-Process#footer

Reply:
also FYI on a per repo level you can enable this https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-autolinks-to-reference-external-resources

id:<num> -> https://ncarb.tpondemand.com/entity/<num>

Reply:
I was thinking about that.

We could turn that on for all repos and pair down the fluff more, perhaps.

Reply:
I wish it was at an org level 😞